_api-reference/cluster-api/cluster-settings.md

```diff
@@ -102,6 +102,7 @@ The following request field parameters are compatible with the cluster API.
 | cluster.routing.allocation.include.<attribute> | Enum | Allocates shards to a node whose `attribute` has at least one of the included comma-separated values. |
 | cluster.routing.allocation.require.<attribute> | Enum | Only allocates shards to a node whose `attribute` has all of the included comma-separated values. |
 | cluster.routing.allocation.exclude.<attribute> | Enum | Does not allocate shards to a node whose `attribute` has any of the included comma-separated values. The cluster allocation settings support the following built-in attributes: <br /> <br /> `_name` – Match nodes by node name. <br /> <br /> `_host_ip` – Match nodes by host IP address. <br /> <br /> `_publish_ip` – Match nodes by publish IP address. <br /> <br /> `_ip` – Match either `_host_ip` or `_publish_ip`. <br /> <br /> `_host` – Match nodes by hostname. <br /> <br /> `_id` – Match nodes by node ID. <br /> <br /> `_tier` – Match nodes by data tier role. |
+| cluster.routing.allocation.shard_movement_strategy | Enum |  Determines the order in which shards are relocated from outgoing to incoming nodes. This setting supports the following strategies: <br /> <br /> `PRIMARY_FIRST` - Primary shards are relocated first, before replica shards. This prioritization may help prevent a cluster's health from going red if the relocating nodes go down during the process. <br /> <br /> `REPLICA_FIRST` - Replica shards are relocated first, before primary shards. This prioritization may help prevent a cluster's health from going red when carrying out shard relocation in a mixed-version, segment-replication-enabled OpenSearch cluster. In this situation, primary shards relocated to OpenSearch nodes of a newer version could try to copy segment files to replica shards on an older version of OpenSearch, which would result in shard failure. Relocating replica shards first attempts to avoid this from happening in multi-version clusters. <br /> <br /> `NO_PREFERENCE` - The default behavior in which order of shard relocation has no importance.
```

Comment:
Under `PRIMARY_FIRST`: "This prioritization may help prevent a cluster's health status from going red if the relocating nodes fail during the process"?

Under `REPLICA_FIRST`: "a cluster's health status"? Also, the last sentence of this description is slightly awkward. Do we mean something like "Relocating replica shards first may help to avoid this in multi-version clusters"?